app/src/main/java/org/worldbank/transport/driver/utilities/RecordFormPaginator.java

```diff
@@ -0,0 +1,296 @@
+package org.worldbank.transport.driver.utilities;
+
+import android.util.Log;
+
+import org.jsonschema2pojo.annotations.Multiple;
+import org.jsonschema2pojo.annotations.PluralTitle;
+import org.jsonschema2pojo.annotations.Title;
+import org.worldbank.transport.driver.activities.RecordFormSectionActivity;
+import org.worldbank.transport.driver.activities.RecordItemListActivity;
+import org.worldbank.transport.driver.models.DriverSchema;
+
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+
+/**
+ * Static methods to support moving around in paginated form sections.
+ *
+ * Each section is one field on the DriverSchema class.
+ * Their order is determined by the JsonPropertyOrder annotation on the DriverSchema class.
+ *
+ * Created by kathrynkillebrew on 1/4/16.
+ */
+public class RecordFormPaginator {
```

Comment:
Sectionator, perhaps? That's only half serious, but 'paginator' calls to mind paginating a long list of results into smaller pages which is different from what this does.